<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPackage</class>
 <widget class="QDialog" name="addPackage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:century gothic;
color:white;
}
QLabel{
font-size:16px;
}
#head{
border-bottom-left-radius: 20px;
border-bottom-right-radius: 20px;
background: qlineargradient(spread:pad, x0:1, y1:0, x2:1, y2:1,stop:0 #6284FF, stop:1 #FF0000);
font-size:24px;
font-weight:bold;
}
#bookingValidate, #durationValidate, #departureValidate{
font-size:12px;
color:red;
background:transparent;
}
QLineEdit{
font-size:16px;
font-weight:bold;
color:black;
background:white;
}
#addPackage,#frame{
background:#282828;
}
#packageDescriptionEdit{
color:black;
background-color:white;
}
QPushButton {
font-size:15px;
font-weight:bold;
border:none;
border-radius:5px;
background: #ff0303;
}
QPushButton:hover {
background:#49ebff;
color:black;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>678</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="head">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>676</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD PACKAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>625</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>625</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="packageNameLabel">
       <property name="text">
        <string>Package Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="packageNameEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="destinationLabel">
       <property name="text">
        <string>Destinations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="destinationEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="hotelLabel">
       <property name="text">
        <string>Hotel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="transportLabel">
       <property name="text">
        <string>Transport</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="hotelEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="transportEdit">
       <property name="placeholderText">
        <string>if none: Not included</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="departureLabel">
       <property name="text">
        <string>Departure</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="bookingStartsLabel">
       <property name="text">
        <string>Booking</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="bookingStartsEdit">
       <property name="placeholderText">
        <string>XX to XX of month</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="departureEdit">
       <property name="placeholderText">
        <string>XX of month</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>661</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="NumOfTravellersLabel">
       <property name="text">
        <string>Number Of Travellers Allowed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="durationLabel">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="durationEdit">
       <property name="placeholderText">
        <string>XX Nights &amp; XX Days</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="NumOfTravellersEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="foodlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Food </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="foodEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>326</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>if none: Not included</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="packageDescriptionEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>separate detail by comma(',')</string>
    </property>
   </widget>
   <widget class="QLabel" name="packageDescriptionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>485</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Package Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pricePerPersonEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>580</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>In USD</string>
    </property>
   </widget>
   <widget class="QLabel" name="pricePerPersonLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>555</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Price Per Person</string>
    </property>
   </widget>
   <widget class="QLabel" name="bookingValidate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>386</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>follow the format &quot;XX to XX of month&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="departureValidate">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>386</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>follow the format &quot;XX of month&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="durationValidate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>466</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>follow the format &quot;XX Nights &amp; XX Days&quot;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
